.mailmap

```diff
@@ -2,4 +2,4 @@ Aamna Lawrence <aamna.lawrence@gmail.com>
 Adam Li <adam2392@gmail.com>
 Victor Xiang <xiangliang3151@gmail.com>
 Victor Xiang <xiangliang3151@gmail.com> <32950290+Nick3151@users.noreply.github.com>
-Yorguin Mantilla <36543115+yjmantilla@users.noreply.github.com>
+Yorguin Mantilla <yjmantilla@gmail.com>
```

Comment:
The `.mailmap` file exists to keep the git history clean, specifically `git shortlog --summary` --> for example when a single author makes commits from different computers or accounts, these commits may show up as if from a different person 

`.mailmap` can be used to merge these "identities" back to how they are supposed to be represented.

In your case, `git shortlog --summary` yields your full name ... and based on what you added to whats_new.rst a while back, I inferred that you want to be represented with just those two names.

Next to your name you can also determine the email address that you want to be the "representation" ... From your edit I assume you want that to be Gmail. To get that right however, you need to modify the syntax a bit.

See this example:

 https://github.com/mne-tools/mne-bids/blob/2f8ef1b7a9830f663c7ae7b5c7b6dc5a54f230e4/.mailmap#L13-L14

So this would be correct:
```suggestion
Yorguin Mantilla <yjmantilla@gmail.com>
Yorguin Mantilla <yjmantilla@gmail.com> <36543115+yjmantilla@users.noreply.github.com>
```